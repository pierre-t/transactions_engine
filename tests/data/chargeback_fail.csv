type,client,tx,amount
deposit,1,1,1.0
deposit,1,2,2.0
chargeback,1,1,
dispute,1,2,
chargeback,1,2,
deposit,2,10,100.0
dispute,2,11,
chargeback,2,11,
dispute,2,10,
chargeback,1,10,
chargeback,2,99,
chargeback,3,10,
