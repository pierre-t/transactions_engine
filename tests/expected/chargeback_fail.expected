client,available,held,total,locked
1,1,0,1,true
2,0,100,100,false